pom.xml

```diff
@@ -52,10 +52,36 @@
 		<jetty.version>9.1.0.v20131115</jetty.version>
 		<servlet-api.version>3.1.0</servlet-api.version>
 		<hibernate.version>4.3.0.Final</hibernate.version>
-		<liquibase.property.file>src/main/resources/liquibase/liquibase.properties</liquibase.property.file>
+		<liquibase.property.file>/target/classes/liquibase/liquibase.properties</liquibase.property.file>
 		<skipLiquibaseRun>false</skipLiquibaseRun>
+
+		<jdbc.url>jdbc:postgresql://postgres:5432/postgres</jdbc.url>
 	</properties>
 	<build>
+	<!-- 
+	This resources block was taken from the Spring parent project.
+	The only change that wanted to make was to include the liquibase.properties file.
+	I carried the others forward just in case I ran into issues.
+	 -->
+    <resources>
+      <resource>
+        <filtering>true</filtering>
+        <directory>src/main/resources</directory>
+        <includes>
+          <include>**/application.yml</include>
+          <include>**/application.properties</include>
+          <include>**/application-no-web.properties</include>
+          <include>**/liquibase.properties</include>

```

Comment:
The moving of files happen automatically. The filtering trick will take properties fromnthe pom and put them into the source code so that by the time they are in target/classes they will have the appropriate props based on the profile that was ran.